tests/monotouch-test/Compression/CompressionStreamTest.cs

```diff
@@ -44,13 +44,17 @@ private static bool compare_buffers (byte[] first, byte[] second, int length)
 		[ExpectedException (typeof (ArgumentNullException))]
 		public void Constructor_Null ()
 		{
+			if (!TestRuntime.CheckXcodeVersion (7, 0))
```

Comment:
_For the future_: there's `TestRuntime.AssertXcodeVersion` as well, which allows you to do this in a single line.